acceptance/acceptance_test.go

```diff
@@ -259,6 +262,14 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	cmd.Dir = tmpDir
 	err = cmd.Run()
 
+	// If a custom server was defined, record all requests made to an output file.
```

Comment:
These are orthogonal features - I might want to record responses with default server or have custom server and no recordings.

Reply:
Done